command/init.go

```diff
@@ -337,6 +337,15 @@ job "example" {
       #   change_signal = "SIGHUP"
       # }
 
+      # The "template" stanza can also be used to create env vars for tasks
```

Comment:
I would say "environment variables" and also mention the fact that the task will be restarted when the env vars change.